python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -346,6 +347,26 @@ async def configure(self, config):
                 DeprecationWarning,
             )
 
+        # Determine the amount of time spent on target so the rotator position
+        # can be optimized. Assumed slew time is 3 minutes,
+        # read+calculation+offset is 20s
+        if self.do_acquire:
+
```

Comment:
I did a few things:
1. Added it as a config parameter... but check the default value handling... i don't like it but don't know how to use "None" as a default value since it isn't a number.
2. made the calculation a method... i suppose it's cleaner but it's not really decreasing the amount of code etc.